docs/source/concepts/calibration/PowderDiffractionCalibration.rst

```diff
@@ -662,13 +659,13 @@ This can be done using the following code
 
 .. code::
 
-   from mantid.simpleapi import (AlignDetectors, LoadDiffCal, LoadEventNexus, LoadInstrument, Rebin)
+   from mantid.simpleapi import (ApplyDiffCal, ConvertUnits, LoadEventNexus, LoadInstrument, Rebin)
    from Calibration.tofpd import diagnostics
 
    LoadEventNexus(Filename='VULCAN_192227.nxs.h5', OutputWorkspace='ws')
    Rebin(InputWorkspace='ws', OutputWorkspace='ws', Params=(5000,-.002,70000))
-   LoadDiffCal(Filename='VULCAN_Calibration_CC_4runs_hybrid.h5', InputWorkspace='ws', WorkspaceName='VULCAN')
-   AlignDetectors(InputWorkspace='ws', OutputWorkspace='ws', CalibrationWorkspace='VULCAN_cal')
+   ApplyDiffCal(InputWorkspace='ws', CalibrationFile='VULCAN_Calibration_CC_4runs_hybrid.h5')
```

Comment:
@wdzhou - I don't have the files to test this change and it's not covered by a doc test, but this should work as it looks like  `ApplyDiffCal` calls `LoadDiffCal` - do you agree? Don't know if you're still working on VULCAN calibration and may have these files anywhere?

Reply:
Did we determine an answer to this question?

Reply:
Looking at the lines of change, this looks identical to what we did in other places.

Wenduo left the project a couple of years ago so don't expect a response from him.